packages/custom-elements/ts_src/Patch/Element.ts

```diff
@@ -179,6 +179,31 @@ export default function (internals: CustomElementInternals) {
     }
   };
 
+  Element.prototype.toggleAttribute = function (
```

Comment:
It looks like the tests are failing because they're being run unconditionally in browsers that don't have native support for `toggleAttribute`, which also reminded me that we shouldn't write `toggleAttribute` if a native one didn't exist before. If you wrap this block in a check for `Native.Element_toggleAttribute`

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/custom-elements/ts_src/Patch/Element.ts#L21-L22

and then skip the root suites in the new tests

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/tests/custom-elements/html/Interface/ChildNode/index.test.html#L49-L58

that should fix these.

Reply:
So we're not loading the platform polyfill for these test?